src/Uno.UI/UI/Xaml/Media/LinearGradiientBrush.faux.cs

```diff
@@ -0,0 +1,120 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+using Uno.UI.Xaml.Media;
+using Windows.Foundation.Collections;
+using Windows.UI;
+
+namespace Microsoft.UI.Xaml.Media;
+
+public partial class LinearGradientBrush
+{
+	private Color? _majorStopColor;
+	private SolidColorBrush? _fauxOverlayBrush;
```

Comment:
Let's put these under `#if __WASM__ || __IOS__ || __MACOS__` so it's more clear what platforms are relevant.

Reply:
I put the whole file in this `#if`, as other targets don't need it at all now